unstructured/ingest/cli/cmds/vectara.py

```diff
@@ -0,0 +1,59 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig
+from unstructured.ingest.connector.vectara import SimpleVectaraConfig, VectaraWriteConfig
+
+
+@dataclass
+class VectaraCliWriteConfig(SimpleVectaraConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--customer-id"],
+                required=True,
+                type=str,
+                help="The Vectara customer-id.",
+                envvar="VECTARA_CUSTOMER_ID",
+                show_envvar=True,
+            ),
+            click.Option(
+                ["--oauth-client-id"],
+                required=True,
+                type=str,
+                help="Vectara OAuth2 client ID.",
+                envvar="VECTARA_OAUTH_CLIENT_ID",
+                show_envvar=True,
+            ),
+            click.Option(
+                ["--oauth-secret"],
+                required=True,
+                type=str,
+                help="Vectara OAuth2 secret.",
+                envvar="VECTARA_OAUTH_SECRET",
+                show_envvar=True,
+            ),
+            click.Option(
+                ["--corpus-name"],
+                required=False,
+                default="vectara-unstructured",
```

Comment:
Rather than use an `unstructured` default, let's make this required and have no default.

Reply:
yep. The default doesn't make any sense...